CHANGELOG.md

```diff
@@ -5,5 +5,6 @@ which are exposed to antikythera instance administrators and/or gear developers.
 
 ---
 
-- In the next minor version:
+- 0.1.1:
```

Comment:
Don't misunderstand "minor" and "patch" versions in semver.
If you are going to release 0.1.1 then rate limiting for async job queues shouldn't be included; make a branch for `0.1.x`.
Or, should we bump to 0.2.0?

Reply:
Ah my mistake.
Let's bump to `0.2.0`, since it was already an internal plan for this week (of course individual antikythera instances can decide when to adopt the change).
Leaving `0.1.x` series (in hex.pm) "uncompilable" is somewhat unkind but at least in current phase it should cause no major backlash.

Reply:
in a771166